Makefile

```diff
@@ -31,4 +31,4 @@ doc:
 	godoc -http=localhost:$(PORT)
 
 check:
-	bash ./check.sh
+	bash ./check.sh
```

Comment:
I guess this change deletes the newline at the end of the line. Could you revert it?

Reply:
Resolved, [0042502](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/0042502d165af4bbfd57b99474f816dd147c4599).

Reply:
Resolved, [bee4be0](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/bee4be09cc55d2003a3c84ff7cdbac923c4185f9).